utils/emissions.go

```diff
@@ -22,21 +22,38 @@ const (
 	perDay   string = "Per Day"
 )
 
-var countryIDs = []string{"UNITED_STATES", "GERMANY", "UNITED_KINGDOM", "AUSTRALIA", "SINGAPORE", "JAPAN", "ICELAND", "NORWAY", "CHINA", "SWEDEN"}
+var countryIDs = []string{"UNITED_STATES", "GERMANY", "UNITED_KINGDOM", "AUSTRALIA", "SINGAPORE", "JAPAN",
+	"ICELAND", "NORWAY", "CHINA", "SWEDEN"}
+
+var networkMap = map[string]string{
+	"camino":     "mainnet",
```

Comment:
Could you please explain why do we have mainnet here and not camino cause i dont understand..? Mainnet is not for avax?

Reply:
this is because immutable insights people have the ids for the camino network like "caminomainnet"
![image](https://user-images.githubusercontent.com/49324893/234609094-febd3941-a9c9-4dcc-b2bd-1ade79c2c604.png)